### ISSUE TITLE
hotbar msg instead title

### ISSUE THREAD

**Issue by RedstoneFuture** [IntellectualSites/PlotSquaredSuggestions#25]
If I activate the title flag the user get infos about the flag with a plot-join. This title I find are to big and would like use the hotbar for the same infos.

**Comment by AbelMG** [comment]
I think there is a plotsquared add-on that makes that already

**Comment by RedstoneFuture** [comment]
@AbelMG Which Addon do you mean?

**Comment by RedstoneFuture** [comment]
@RemyyB I think a greeting flag or - better - a separate title flag with a short message is a great idea too.

**Comment by RedstoneFuture** [comment]
Its maybe is simple change. The title to large for the plot information.

**Comment by boy0001** [comment]
You can just disable the titles in the settings and set a default `greeting` flag in the worlds.yml.

**Comment by RedstoneFuture** [comment]
@boy0001 Yes, I disabled this already since the beginning. But I think this plot-information as hotbar message would be good for server without a plot-scoreboard. You get very fast the plot-information If you fly over the plot and see it in the hotbar. I think this fast way was the reason the added the titles for a long time ago. But yes, the titles are very large and maybe to large for the minecraft window.

**Comment by AeSix** [comment]
I have to agree with @RedstoneFuture 
Either changing the presentation to subtitle (which appears just above the hotbar) or a config option would be nice.  

I have had incidents where titles overflow the visual bounds of the screen, and so is cut off.  This happens with PlotSquared with default window size and more than 6 or 7 characters.  It's rather annoying.  

This could also allow for persistent plot info (per player toggle) without being obnoxious.  There's been many times when I wish I had that info at a glace, for various administrative duties.

**Comment by N0tMyFaultOG** [comment]
How about not removing any of them? The presentation could be defined by a boolean config option to appear over the hotbar or use the titles.
